integration-test/langfuse-integration-fetch.spec.ts

```diff
@@ -641,4 +642,65 @@ describe("Langfuse (fetch)", () => {
       expect(fetchedGeneration.data.output).toEqual("MASKED");
     });
   });
+  it("replace media reference string in object", async () => {
+    const mockTraceName = "test-trace-with-audio" + Math.random().toString(36);
+    const mockAudioBytes = fs.readFileSync("./static/joke_prompt.wav"); // Simple mock audio bytes
+
+    const trace = langfuse.trace({
+      name: mockTraceName,
+      metadata: {
+        context: {
+          nested: new LangfuseMedia({
+            base64DataUri: `data:audio/wav;base64,${Buffer.from(mockAudioBytes).toString("base64")}`,
+          }),
+        },
+      },
+    });
+
+    await langfuse.flushAsync();
+    const res = await axios.get(`${LANGFUSE_BASEURL}/api/public/traces/${trace.id}`, { headers: getHeaders() });
+
+    expect(res.data).toMatchObject({
+      id: trace.id,
+      name: mockTraceName,
+      metadata: {
+        context: {
+          nested: expect.stringMatching(/^@@@langfuseMedia:type=audio\/wav\|id=.+\|source=base64_data_uri@@@$/),
+        },
+      },
+    });
+
+    const mediaReplacedTrace = await LangfuseMedia.resolveMediaReferences({
+      obj: res.data,
+      langfuseClient: langfuse,
```

Comment:
could we add this as a utility method on the langfuse client itself? The method needs to make api calls, thus adding it to the instantiated sdk client would make sense

Example

```
traces = langfuse.fetchTraces()
tracesWithInlineMedia = traces.data.map(langfuse.resolveMediaReferences)
```

Reply:
Added! 👍🏾